### DISCUSSION TITLE
useFieldArray and Controller not in Remix Hook Form

### DISCUSSION THREAD

**Discussion by princemuel** [Code-Forge-Net/remix-hook-form#63] [Q&A] [answered]
Hi there. Please I can't find the useFieldArray hook and Controller from react-hook-form.

**Comment by AlemTuzlak** [comment, marked as answer]
@princemuel Remix hook form doesn't re-export those two, you can use the ones from react-hook-form without any problems

**Reply by princemuel** [reply]
Yeah, sorry. I'd already figured it out and forgot to cancel the question. Thanks tho' 💯
